src/CovidAppointmentTable.js

```diff
@@ -39,9 +39,13 @@ export function sortAndFilterData(
     { sortKey, sortAsc },
     onlyShowAvailable
 ) {
+    const tooStaleMinutes = 60;
```

Comment:
could you bring this to the top of the file with a comment? for visibility and clarity

Reply:
Added comment at top of the file. Also commented the code in StaleDataIndictator.js